src/System.Net.Http/tests/UnitTests/HPack/HPackRoundtripTests.cs

```diff
@@ -0,0 +1,214 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http.HPack;
+using System.Net.Http.Headers;
+using System.Text;
+using Xunit;
+
+namespace System.Net.Http.Unit.Tests.HPack
```

Comment:
Is this the same namespace naming convention used elsewhere in the project?

Reply:
Yes